npm start error fix

I was obtaining the following error after `npm start`
```

ERROR in /Users/xyz/code/front-end/angular2/todo-angular2-firebase/node_modules/@types/jasmine/index.d.ts
(37,52): error TS1005: '=' expected.

ERROR in /Users/xyz/code/front-end/angular2/todo-angular2-firebase/node_modules/@types/jasmine/index.d.ts
(37,38): error TS2371: A parameter initializer is only allowed in a function or constructor implementation.

ERROR in /Users/xyz/code/front-end/angular2/todo-angular2-firebase/node_modules/@types/jasmine/index.d.ts
(37,46): error TS2304: Cannot find name 'keyof'.
```
I fixed by modifying line 45 of package.json to the following

`"@types/jasmine": "2.5.36"`

This may address incompatibility with some typescript versions.

**EDIT:** incompatibility issue discussed at http://stackoverflow.com/questions/42182340/angular-2-build-error-expected-in-index-d-ts

--- a/package.json
+++ b/package.json
@@ -75,7 +75,7 @@
     "ts-helpers": "~1.1.2",
     "ts-loader": "~0.9.5",
     "tslint": "~3.15.1",
-    "typescript": "~2.0.6",
+    "typescript": "~2.2.1",
     "typings": "~2.0.0",
     "webpack": "2.1.0-beta.22",
     "webpack-dev-server": "2.1.0-beta.4",

